app/terminal/terminal.tsx

```diff
@@ -72,6 +75,27 @@ export default class TerminalComponent extends React.Component<TerminalProps, Te
   }
 
   handleDownloadClicked() {
+    if (typeof this.props.invocationId === "string") {
+      rpcService.service
+        .getEventLogChunk(
+          new eventlog.GetEventLogChunkRequest({
```

Comment:
nit: while this RPC is loading, we should show a loading spinner so the user gets active feedback that we're working on the request, in case the log is particularly large. Also, if there's an error in the RPC, we can use `errorService` from `app/errors/error_service` to show a temporary error banner.

Something like this should probably work:

```
this.setState({ isLoadingFullLog: true });
rpcService.service.getEventLogChunk(...)
    .then(...)
    .catch((e) => errorService.handleError(BuildBuddyError.parse(e)))
    .finally(() => this.setState({ isLoadingFullLog: false });
```

and when rendering (note the `disabled` state and also the `Spinner` from `app/components/spinner/spinner`):

```
<button ... disabled={this.state.isLoadingFullLog}>
  {this.state.isLoadingFullLog ? <Spinner /> : <img ... />}
</button>
```

Might need some CSS tweaks to make the spinner the same size as the download button, as well as making sure the spinner works in both light and dark terminal theme.